plugins/conference/services/ConferenceService.php

```diff
@@ -58,7 +58,14 @@ public function allocateConferenceRoomImpl($entryId)
 		$existingConfRoom = $this->findExistingConferenceRoom($entryId);
 		if ($existingConfRoom)
 			return $existingConfRoom;
-
+		
+		$partner = $this->getPartner();
+		$numOfConcurrentRtcStreams = EntryServerNodePeer::retrieveByPartnerIdAndServerType($this->getPartnerId(), ConferencePlugin::getCoreValue('EntryServerNodeType', ConferenceEntryServerNodeType::CONFERENCE_ENTRY_SERVER ));
+		if ($numOfConcurrentRtcStreams >= $partner->getMaxLiveRtcStreamInputs())
```

Comment:
if getMaxLiveRtcStreamInputs returns null because it was never set we should have a default
have a look at LiveStreamService.php:212